src/components/ChartPanel/chartCalculations.js

```diff
@@ -0,0 +1,29 @@
+export const calculateRateChange = data => {
+	if (data.length < 2) return 0;
+	const newPrice = data[0].rate;
+	const oldPrice = data[data.length - 1].rate;
+	const percentageChange = (newPrice - oldPrice) / Math.abs(oldPrice);
```

Comment:
[np] why `abs` the `oldPrice`?

Reply:
Hmm yeah I'm not sure. A price won't be negative so it shouldn't be here.

Reply:
I just removed it.